SceneMetadata/ExampleSceneMetadata.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Unity.Labs.SuperScience
+{
+    public class ExampleSceneMetadata : ScriptableObject
```

Comment:
Summary comment, even just call out that this is the data object that is created by the modification processor